app/translations/en.json

```diff
@@ -111,7 +111,7 @@
   "app.common.showLess": "Show less {value}",
   "app.common.poweredByTelos": "Powered by {image}",
   "app.common.poweredByPeeranha": "Powered by {image}",
-  "app.common.termsOfTelos": "Telos Operating Agreement",
+  "app.common.termsOfTelos": "Operating Agreement",
```

Comment:
`termsOfTelos` translation key should be changed to a new one

Reply:
changed: https://github.com/peeranha/peeranha-web/pull/650/commits/fcf50b44b1470726b3dd53763fca278224a03a17